### PR TITLE
Improve DCAT harvesting

### PR DESCRIPTION
**Pull request by alexarobu**
This improves the general XML formatting of the file and adds a link back to the Spatial Hub

### CODE REVIEW

File: src/main/plugin/iso19139.gemini23/convert/fromSPARQL-DCAT.xsl [hunk 1]
```diff
@@ -527,12 +528,44 @@
...
-          </xsl:if>
+                  </xsl:if>
+
+                  <!-- Link to CKAN record -->
+
+                  <gmd:onLine>
+                    <gmd:CI_OnlineResource>
+                      <gmd:linkage>
+                        <gmd:URL>
+                          <xsl:text>https://data.spatialhub.scot/dataset/</xsl:text>
```
Comment (archaeogeek) [inline src/main/plugin/iso19139.gemini23/convert/fromSPARQL-DCAT.xsl]: Not sure we should be hard-coding this in the standard Gemini branch

File: src/main/plugin/iso19139.gemini23/convert/fromSPARQL-DCAT.xsl [hunk 2]
```diff
@@ -527,12 +528,44 @@
...
+                      </gmd:protocol>
+                      <gmd:name>
+                        <gco:CharacterString>
+                          <xsl:value-of select="gn-fn-sparql:getObject($root,
+                                                'http://purl.org/dc/terms/title',
+                                                $resourceUri)/sr:literal"/>
+                        </gco:CharacterString>
+                      </gmd:name>
+                      <gmd:description>
+                        <gco:CharacterString>A link to the CKAN metadata record on the Spatial Hub.</gco:CharacterString>
```
Comment (archaeogeek) [inline src/main/plugin/iso19139.gemini23/convert/fromSPARQL-DCAT.xsl]: Same, this probably shouldn't be hard-coded in the standard Gemini file

**Review by archaeogeek — CHANGES_REQUESTED** [review]
From what I can tell, this is hard-coding the spatialhub address in the file (unless I mistook what's going on). I'd be reluctant for that to go into the standard plugin- so you might want to think about a way of doing this within the scotgov code instead- even if it's just to overwrite the standard one when you deploy

**Comment by alexarobu** [comment]
@archaeogeek fair enough, I don't know why I didn't think it should have been more generic, how about now?

**Review by archaeogeek — APPROVED** [review]
Great! Happy to approve

**Comment by alexarobu** [comment]
> Great! Happy to approve

Super, thanks @archaeogeek :pray:

**Pull request merged**